packages/entities/entities-shared/src/components/entity-base-table/EntityBaseTable.vue

```diff
@@ -364,18 +363,6 @@ const { setTablePreferences, getTablePreferences } = useTablePreferences()
 
 const tablePreferences = ref<TablePreferences>(getTablePreferences(cacheId.value))
 
-const combinedInitialFetcherParams = computed((): Partial<FetcherParams> => {
-  // Pass the preferencesStorageKey regardless; if no entry is found, it will return the default
-  const userTablePreferences = getTablePreferences(cacheId.value)
-
-  // Return the props.initialFetcherParams, appending any stored user preferences
-
-  return {
-    ...props.initialFetcherParams,
-    ...userTablePreferences,
-  }
```

Comment:
@mptap or @kaiarrowood (not sure who implemented this piece) - I don't see a need to pass table preference in initial params prop, am I missing something?

Reply:
We need it for the default `page` / `pageSize` which is also part of the preferences

Reply:
What Kai said ^

Reply:
Got it, adding back. I'm guessing that's also why the tests are failing